frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -398,6 +411,12 @@
       <DetailItem title={$t('project_page.project_code')} text={project.code} copyToClipboard={true} />
       <DetailItem title={$t('project_page.created_at')} text={$date(project.createdDate)} />
       <DetailItem title={$t('project_page.last_commit')} text={$date(project.lastCommit)} />
+      {#if loadingRepoSize}
+        <Loader loading={loadingRepoSize} size="loading-xs" />
+      {:else}
+        <!-- TODO: Might be nice to create a size formatter that will convert to megabytes/gigabytes as appropriate -->
```

Comment:
What if we more or less just copy one of these:
https://github.com/Datamart/bytes-formatter/blob/b8722b3063c0c1a9299763c61d3508528da651f7/index.js#L30
https://gist.github.com/zentala/1e6f72438796d74531803cc3833c039c

Reply:
The `Math.log` one is clever, but I suspect it does a lot more work than the version that divides by 1024. It does have no if branches at all, so it's got that going for it.

But since nearly all of our repos are in the megabyte range, I'm inclined to just divide once by 1024 and print the results. The few repos that are 800 kB or so can show up as 0.8 MB. Why just divide once by 1024 unconditionally? No if branches, no [branch prediction failure](https://stackoverflow.com/a/11227902).

So on the YAGNI principle, I'll defer making a general-purpose `size` formatter until we need some unit other than MB for everything.

Reply:
Megabyte formatter added in commit c5a4cae3. Screenshots in PR description show what it looks like now.